### PR TITLE
Cw20 logo spec

### PR DESCRIPTION
**Pull request by ethanfrey**
Add a new optional extension to the cw20 spec to allow for managing more marketing-related metadata on-chain. Including links to the projects site as well as token logo.

Please review this if there is more info needed here, to suggest better wording, to point out some security issue, or to comment on DownloadLogoResponse compatibility with React/Vue.

### CODE REVIEW

File: packages/cw20/src/logo.rs
Comment (hashedone) [inline packages/cw20/src/logo.rs:10]: I don't think it is insecure in anyway from blockchain POV - we are not using it internally. The question is if it is possible to use it on external side, and I am not aware about remote execution method using embedded images. And even if it is a case, I think it is webpage who is using this URL responsibility to ensure this image is trusted. My concern is, that keeping images in blockchain/external servers allows abuses with usage of logo, as it is more difficult to moderate this if it lies in blockchain than if it is on owner server, but this relates to both methods of uploading image.

Anyway - whatever final decision is, pleas remember to remove this comment before merging.

**Comment by ethanfrey** [comment]
> I don't see any actual implementation on storing/retrieving those data - are those meant to be just interfaces?

Yes, this is just interfaces. The next task #371 is to actually implement them for cw20-base. (And maybe add helpers.rs methods in cw20)

**Review by hashedone — APPROVED** [review]
Then looks good. I don't merge it myself in case you want anyone else to check for those potential security issues.

**Pull request merged**
